pkg/handler/collector/collector.go

```diff
@@ -108,8 +118,10 @@ func Collect(ctx context.Context, emitter Emitter, handleErr ErrHandler) error {
 	}
 	for len(docChan) > 0 {
 		d := <-docChan
+		addChildLogger(logger, d)
```

Comment:
@nathannaveen fixed the issue, the child logger was not getting initiated here.

Reply:
Oh, thank you @pxp928!!